Source/BSGConnectivity.m

```diff
@@ -50,21 +50,19 @@ @interface BSGConnectivity ()
 
 @property(nonatomic, assign) SCNetworkReachabilityRef reachabilityRef;
 @property(nonatomic, strong) dispatch_queue_t serialQueue;
-@property(nonatomic) CallbackBlock callbackBlock;
+@property(nonatomic, copy) CallbackBlock callbackBlock;
```

Comment:
Just as `connectivityChangeBlock`, the `callbackBlock` set should be copied from stack to heap.